.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "google"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 13,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+    }
```

Comment:
Do we want to open issues for discussing to add new rules? It's kind of fun to create our own coding guides :)

Reply:
sure, I'll open one in a minute